server/controllers/resetpassword.js

```diff
@@ -0,0 +1,39 @@
+import notification from '../helpers/notifications';
+import Authenticate from '../helpers/auth';
+import model from '../models';
+
+const { User } = model;
+
+const { hashPassword } = Authenticate;
+
+const ResetPasswordController = {
+  async updatePassword(req, res) {
+    const { userObj } = req.user;
```

Comment:
But have a different function to sign the user's email to avoid this naming confusion